### PR TITLE
[MIG] account_asset_batch_compute: Migration to 13.0

### PR DESCRIPTION
**Pull request by albariera**
Adapted to new queue logic.

### CODE REVIEW

**Comment by etobella** [comment]
@JordiBForgeFlow You might be interested on this

File: account_asset_batch_compute/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Adrien Peiffer
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
```
Comment (JordiBForgeFlow) [inline account_asset_batch_compute/readme/CONTRIBUTORS.rst]: Can you change to jordi.ballester@forgeflow.com

File: account_asset_batch_compute/tests/test_account_asset_batch_compute.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
+# Copyright 2016-19 ACSONE SA/NV
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
```
Comment (JordiBForgeFlow) [inline account_asset_batch_compute/tests/test_account_asset_batch_compute.py]: Change to ForgeFlow, S.L.

File: account_asset_batch_compute/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+~~~~~~~
+
+* ACSONE SA/NV
+* Eficent
+
+Contributors
+~~~~~~~~~~~~
+
+* Adrien Peiffer
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
```
Comment (JordiBForgeFlow) [inline account_asset_batch_compute/README.rst]: Change to jordi.ballester@forgeflow.com

**Review by JordiBForgeFlow — APPROVED** [review]
Thanks @albariera! Looks good to me. Since we have renamed Eficent -> ForgeFlow, can you please take the opportunity to change it?

**Comment by albariera** [comment]
@JordiBForgeFlow comments attended

**Review by etobella — APPROVED** [review]

**Pull request merged**